mmv1/third_party/terraform/tests/resource_pubsub_subscription_test.go

```diff
@@ -118,6 +118,31 @@ func TestAccPubsubSubscription_push(t *testing.T) {
 	})
 }
 
+func TestAccPubsubSubscription_noWrap(t *testing.T) {
+	t.Parallel()
+
+	topicFoo := fmt.Sprintf("tf-test-topic-foo-%s", acctest.RandString(t, 10))
+	subscription := fmt.Sprintf("tf-test-sub-foo-%s", acctest.RandString(t, 10))
+	saAccount := fmt.Sprintf("tf-test-pubsub-%s", acctest.RandString(t, 10))
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckPubsubSubscriptionDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccPubsubSubscription_noWrap(topicFoo, saAccount, subscription),
+			},
+			{
+				ResourceName:      "google_pubsub_subscription.foo",
+				ImportStateId:     subscription,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
```

Comment:
Can you please add more steps to test updating the filed `write_metadata` from true to false? Thanks.